angular.json

```diff
@@ -469,9 +469,12 @@
             },
             "architect": {
                 "build": {
-                    "builder": "@angular-devkit/build-angular:browser",
+                    "builder": "@angular-builders/custom-webpack:browser",
                     "outputs": ["dist/demo"],
                     "options": {
+                        "customWebpackConfig": {
+                            "path": "scripts/webpack/assets.config.ts"
```

Comment:
why not 
`projects/demo/webpack.config.ts`

Reply:
I will move it 👌